website/src/components/Buttons/LikertButtons.tsx

```diff
@@ -0,0 +1,45 @@
+import { Button, SimpleGrid } from "@chakra-ui/react";
+import { PropsWithChildren, ReactNode } from "react";
+
+export const LikertButtons = ({
+  isDisabled,
+  options,
+  value,
+  onChange,
+}: PropsWithChildren<{
+  isDisabled: boolean;
+  options: ReactNode[];
+  value: number;
+  onChange: (number) => void;
+}>) => {
+  return (
+    <SimpleGrid
+      aria-roledescription="radiogroup"
+      data-cy-complete={value !== null}
+      columns={options.length}
+      spacing={[1, 4]}
+    >
+      {options.map((option, idx) => {
+        const indexValue = idx / (options.length - 1);
+        return (
+          <Button
+            aria-roledescription="radio"
+            aria-checked={indexValue === value}
+            key={idx}
+            onClick={() => {
+              onChange(indexValue === value ? null : indexValue);
+            }}
+            isDisabled={isDisabled}
+            isActive={indexValue === value}
+            textOverflow="ellipsis"
+            paddingLeft="0"
+            paddingRight="0"
```

Comment:
`px='0'`?

Reply:
In the end I decided the padded wasn't required. In general I find the px and similar short names quite cryptic as someone mostly familiar with traditional css, would paddingX have been acceptable in this case or are we trying to be consistent about using the short names?